components/automate-ui/src/app/pages/user-details/user-details.component.ts

```diff
@@ -173,7 +178,7 @@ export class UserDetailsComponent implements OnInit, OnDestroy {
   }
 
   public updateFullName(): void {
-    const name = this.editForm.get('fullName').value;
+    const name = this.editForm.get('fullName').value.trim();
```

Comment:
Does this interact in some way with the new "mark as pristine" feature? Could this look like a dirty field because there was some space trimmed...?

Reply:
Yes, but no. 😉 That is, the pristine/dirty machinations occur _while_ the user is editing. Once the user presses "save", _then_ the field is immediately trimmed before being sent to the back-end, so we store a correct value. (I believe that in every instance in our code the form is then closed so the now out-of-sync on-screen value evaporates.)

But, in one sense, this is actually creating a case that the pristine/dirty mechanism should have caught... but the consequence is negligible: if the user just adds some trailing whitespace we are simply re-saving the thing with the same name as before.